src/platform/forms/tests/migrations.unit.spec.js

```diff
@@ -63,6 +63,7 @@ describe('form migrations:', () => {
     const allFormIds = Object.keys(schemas).filter(formId => !excludedForms.has(formId));
     const reformattedIds = mappedIds.slice(0);
     reformattedIds.splice(0, 1, '1010ez');
+    reformattedIds.splice(-2, 1, 'COMPLAINT-TOOL');
```

Comment:
@jholton @erikphansen @lihanli, we are doing some hacky things here to accommodate for the uppercase formId (since the backend schema is lowercase), should we make the schema name uppercase as well? Also, should we replace all instances of "complaint" with "feedback", on the frontend and backend?

Reply:
This work is now captured in [11851](https://github.com/department-of-veterans-affairs/vets.gov-team/issues/11851).